### DISCUSSION TITLE
A Pythia package index?

### DISCUSSION THREAD

**Discussion by DamienIrving** [ProjectPythia/projectpythia.github.io#129] [Ideas]
Similar to the resource gallery of educational materials, I wonder whether Pythia should host a package index? I'm thinking of something similar to the [PyAOS package index](https://pyaos.github.io/packages/), but for all of geoscience. Similar to the PyAOS package index it would probably be a good idea to clearly define the scope of the package listing and also provide a general overview of the Python geoscience stack to orient people (e.g. here's the PyAOS [scope](https://github.com/PyAOS/pyaos.github.io/blob/master/README.md#scope) and [stack overview](https://pyaos.github.io/stack/) as examples). 

Having a package index might also be a nice way to avoid clogging up the resource gallery with links to package homepages (e.g. dask, geopandas, GeoCAT, etc). (i.e. those links would be available in the package index instead)

**Comment by dcamron** [comment]
Thanks for the suggestion @DamienIrving! We chat briefly about this at today's Pythia-Education meeting and seemed to mostly settle on not being ready to create and maintain this sort of authoritative index like this just yet. We try to provide homepage/installation instructions as links within relevant sections of our foundations material for now, and I think we will have some sort narrative overview of the ecosystem that we plan to cover also within foundations. Still an open discussion for consideration for sure!
